### ISSUE TITLE
Data Detail

### ISSUE THREAD

**Issue by 152056212** [gotu0000/Apollo#1]
Hello, we are very interested in your project, is it convenient to provide your dataset?

**Comment by gotu0000** [comment]
Yeah ! I am working on making a benchmark dataset for good comparison
matric. Is it fine if I give it to you once I finish working on it?
